Turn back on the stem darkening for CFF fonts by default
--- freetype-2.6.2/src/cff/cffobjs.c.orig	2015-11-28 10:08:52.000000000 -0800
+++ freetype-2.6.2/src/cff/cffobjs.c	2015-12-03 15:04:59.202999144 -0800
@@ -1052,7 +1052,7 @@
     driver->hinting_engine = FT_CFF_HINTING_ADOBE;
 #endif
 
-    driver->no_stem_darkening = TRUE;
+    driver->no_stem_darkening = FALSE;
 
     driver->darken_params[0] = CFF_CONFIG_OPTION_DARKENING_PARAMETER_X1;
     driver->darken_params[1] = CFF_CONFIG_OPTION_DARKENING_PARAMETER_Y1;
